lib/foreman_maintain/utils/service.rb

```diff
@@ -0,0 +1,14 @@
+require 'foreman_maintain/utils/service/abstract'
+require 'foreman_maintain/utils/service/systemd'
+require 'foreman_maintain/utils/service/remote_db'
+
+module ForemanMaintain::Utils
+  def self.system_service(name, priority, options = {})
+    db_feature = options.fetch(:db_feature, nil)
+    if name =~ /^(postgresql|.*mongod)$/ && db_feature && db_feature.local? == false
```

Comment:
Nitpick: `!db_feature.local?`. Checking on `== false` is not covering the potential case, where `local?` would return nil, which is also considered false. But the main reason to fix are the conventions.